cmpv2/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "cmpv2"
+version = "0.0.0"
+description = """
+Pure Rust implementation of the Certificate Management Protocol (CMP) as
+described in RFC 4210
+"""
+authors = ["RustCrypto Developers"]
+license = "Apache-2.0 OR MIT"
+repository = "https://github.com/RustCrypto/formats/tree/master/cmpv2"
+categories = ["cryptography", "data-structures", "encoding", "no-std"]
+keywords = ["crypto"]
+readme = "README.md"
+edition = "2021"
+rust-version = "1.65"
+
+[dependencies]
+der = { version = "0.7.0-pre", features = ["oid", "derive", "alloc"], path = "../der" }
+x509-cert = { version = "0.2.0-pre", path = "../x509-cert" }
+spki = { version = "0.7.0-pre", path = "../spki" }
+flagset = { version = "0.4.3" }
+crmf = {version = "0.0", path = "../crmf" }
+
+[dev-dependencies]
+const-oid = { version = "0.10.0-pre", path = "../const-oid" }
+hex-literal = "0.3"
+#reqwest = "0.11.13"
+#tokio = { version = "1.19.2", features = ["full", "time", "rt-multi-thread"] }
+#tokio-test = {version = "0.4.2"}
```

Comment:
Can you remove these if they're (currently) unused?

Reply:
Done. Also removed the commented out test case that these were associated with.